app/styles/pages/public-event.scss

```diff
@@ -104,3 +104,12 @@
     margin-top: 15px;
   }
 }
+
+.loc-map > .g-map {
```

Comment:
This is still not changed. Please pay more attention to detail. Don't miss things like this.